cypress/locators/portrait/index.js

```diff
@@ -3,3 +3,4 @@ import { PORTRAIT_PREVIEW, PORTRAIT_INITIALS } from "./locators";
 // component preview locators
 export const portraitPreview = () => cy.get(PORTRAIT_PREVIEW);
 export const portraitInitials = () => cy.get(PORTRAIT_INITIALS);
+export const portraitAlt = () => cy.get(PORTRAIT_PREVIEW).find("img");
```

Comment:
```suggestion
export const portraitImage = () => cy.get(PORTRAIT_PREVIEW).find("img");
```
In my opinion I would rename this, since we are getting an `img` element with this locator not the `alt` text itself, just to make it clearer what the locator returns when reading the code.